internal/conf/configuration.go

```diff
@@ -322,6 +322,7 @@ type ProviderConfiguration struct {
 	SlackOIDC               OAuthProviderConfiguration     `json:"slack_oidc" envconfig:"SLACK_OIDC"`
 	Twitter                 OAuthProviderConfiguration     `json:"twitter"`
 	Twitch                  OAuthProviderConfiguration     `json:"twitch"`
+	VercelMarketplace       OAuthProviderConfiguration     `json:"vercel_marketplace" envconfig:"VERCEL_MARKETPLACE"`
```

Comment:
```suggestion
	VercelMarketplace       OAuthProviderConfiguration     `json:"vercel_marketplace" split_words:"true"`
```

No need for a special envconfig. It should be configurable via `GOTRUE_EXTERNAL_VERCEL_MARKETPLACE...`